### ISSUE TITLE
Emoji modifiers are not rendered properly

### ISSUE THREAD

**Issue by MichaelJDK** [niklasvh/html2canvas#1813]
### Bug reports:
Emoji modifiers such as male emojis (🤷🏾‍♂️) are not rendered properly:
![image](https://user-images.githubusercontent.com/24390862/56453671-17d1d980-6346-11e9-89f9-616b6f8df4ea.png)

jsfiddle: http://jsfiddle.net/8gf1jbsh/1/ 

### Specifications:

 * html2canvas version tested with:
 * Browser & version: Firefox 66.0.3 (64-bit) and Chrome 73.0.3683.103 
 * Operating system: Windows 10

**Comment by MichaelJDK** [comment]
@niklasvh any news on this bug?

**Comment by thisispete** [comment]
similar problem with red heart emoji  
![2020-02-04](https://user-images.githubusercontent.com/677565/73797533-2c605080-4765-11ea-800a-cb711c011d3b.jpg)

**Comment by thisispete** [comment]
woah.. I have no idea why this works but wrapping the emoji in a span makes them render correctly?!
```
onclone: (doc) => {
        let text = doc.querySelector('.myelement').innerHTML;
        text = text.split('❤️').join('<span>❤️</span>');
        doc.querySelector('.myelement').innerHTML = text;
```

**Comment by niklasvh** [comment]
Fixed with 1.1.5 http://jsfiddle.net/mpdjx86q/
